<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: #007BFF;
    color: white;
    padding: 5px;
    border-radius: 5px;
    font-weight: bold;
}

QLineEdit, QTextEdit {
    border: 1px solid #8D8D8D;      /* stroke abu-abu */
    border-radius: 5px;
    padding: 6px;
    font-size: 14px;
    background-color: #F9F9F9;
    color: #333;
}

/* Efek saat mouse hover */
QLineEdit:hover, QTextEdit:hover {
    border: 2px solid #5A5A5A;      /* warna sedikit lebih gelap saat hover */
}

/* Efek saat widget aktif/fokus */
QLineEdit:focus, QTextEdit:focus {
    border: 2px solid #007BFF;      /* biru terang saat fokus */
    background-color: #FFFFFF;
}

QMainWindow {
    background-color: #E9E9E9;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="Catatan">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>621</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Judul">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="tanggalCatatan">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Judul</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tanggal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Hapus">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>640</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hapus</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExportCsv">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>640</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Export CSV</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Kategori">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Status">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabelCatatan">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>621</width>
      <height>161</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kategori</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Status </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Simpan">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tambahkan ke database</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Pencarian">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Pencarian</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string> :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Edit">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>640</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAss"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSaveAss">
   <property name="text">
    <string>Save Ass</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
